server/app/mutations/grid_services/update.rb

```diff
@@ -41,6 +46,8 @@ def validate
     end
 
     def execute
+      self.grid = self.grid_service.grid
```

Comment:
sure, I'll refactor it so. clearer that way as it follows the same pattern with other helper methods.